docs/api/proposed_endpoints.md

```diff
@@ -356,17 +356,20 @@ Name               | Data type     | Description
 -------------------|---------------|---------------------
 id                 | UUID          | Unique ID for the proposal
 title              | text          | Title of proposal
-speaker            | text          | Speaker for the talk
-kind               | text          | Type of proposal like talk, dev sprint, workshop
+speaker            | UUID          | Foreign Key to `User` Model
+kind               | text          | Type of proposal like `talk`, `dev sprint`, `workshop` or `lightning_talk`
 level              | text          | Level of proposal beginner, intermediate, advanced
-duration           | text          | Duration of talk, sprint or workshop 
+duration           | text          | Duration of talk, sprint, workshop or lightening talk
 abstract           | text          | Abstract of the proposal
 description        | text          | Description of the proposal
 submitted_at       | datetime      | Time of submission of proposal
 approved_at        | datetime      | Time of approval
 modified_at        | datetime      | Time of modification
 status             | text          | Status of proposal like `retracted`, `accepted`, `unaccepted`, `submitted`, etc.
 
+__Note__:
+- For `lightning_talk`, duration will be fixed like `00:05:00`, etc. which would be configuration through settings.
+- For `lightning_talk`, `level` of proposal is like speaker's experience which can be `beginner`, `intermediate` or `advanced`.
```

Comment:
A person can have a different experience in different domains. You cannot assume it as constant. Also, anyone from an audience can be a speaker for lightning talks.